### PR TITLE
update starRating

### PR DESCRIPTION
**Pull request by IgorPopach**

### CODE REVIEW

File: server/routes/rating/rating.js [hunk 1]
```diff
@@ -12,39 +12,39 @@ const handler = {
...
+    };
   }
-  
+
 };
 
 router.use(authenticated);
 router.get('/rating', handler.ratingList);
-router.post('/:type/rating', handler.crateRating);
+router.post('/:entity_type/:entity_id/rating', handler.crateRating);
```
Comment (yodeco) [inline server/routes/rating/rating.js:48]: Better use it with authentification check, like router.post('/:type/rating', authenticated, handler.crateRating);

File: client/src/components/UI/StarRating/StarRating.js [hunk 2]
```diff
@@ -1,21 +1,51 @@
-import React from "react";
+import React, { useState } from "react";
+import axios from "axios";
 
 const starRating = props => {
-    let stars = [];
-    for (let i = 0; i < 5; i++) {
-      let klass = "star-rating__star";
```
Comment (yodeco) [inline client/src/components/UI/StarRating/StarRating.js:6]: maybe className or somehow else? To name via transliteration not ok

Comment (IgorPopach) [reply client/src/components/UI/StarRating/StarRating.js:6]: done

File: server/routes/rating/rating.js (hunk 1 above)
Comment (IgorPopach) [reply server/routes/rating/rating.js:48]: router.use(authenticated);

**Review by yodeco — APPROVED** [review]

**Review by Blager89 — APPROVED** [review]

**Pull request merged**